Dockerfile

```diff
@@ -49,36 +47,51 @@ RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then apt-get -y --no-install-r
     && apt-get clean \
     && rm -rf /var/lib/apt/lists/* ; fi
 
-# Update pip and install requirements
-COPY requirements.txt dev-requirements.txt mssql-requirements.txt ./
-RUN pip install -U pip  \
-    && pip --no-cache-dir install -r requirements.txt -r dev-requirements.txt \
-    && if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then pip --no-cache-dir install -r mssql-requirements.txt ; fi
+#########################
+## Python Dependencies ##
+#########################
+COPY mssql-requirements.txt .
+RUN if [ "$SKIP_MSSQL_INSTALLATION" != "true" ] ; then pip --no-cache-dir install -r mssql-requirements.txt ; fi
+
+COPY dev-requirements.txt .
+RUN pip install -U pip --no-cache-dir install -r dev-requirements.txt
 
-# Copy in the application files and install it locally
+COPY requirements.txt .
+RUN pip install -U pip --no-cache-dir install -r requirements.txt
```

Comment:
my thinking here was that this would potentially speed up the build, given the following logic that may or may not be wrong:
`requirements.txt` is likely to change more often than `dev-requirements.txt` so by loading the one that changes less often first (`dev-requirements.txt`) we can avoid busting the cache for those as often. It looked like the current method had to reinstall _all_ of the requirements whenever either file changed

Reply:
I did some bench marking and build times between main and this branch are nearly identical so with that I think this change is fine.